swagger-spec/logs/definition.yaml

```diff
@@ -0,0 +1,68 @@
+# Log Model Definition
```

Comment:
Can you update this definition to match the new definition style? Preprint example [here](https://github.com/caseyrollins/swagger-ui/blob/master/swagger-spec/preprints/definition.yaml).